src/gcp/credential.rs

```diff
@@ -123,6 +123,7 @@ pub struct ServiceAccountKey(RsaKeyPair);
 
 impl ServiceAccountKey {
     /// Parses a pem-encoded RSA key
+    #[expect(clippy::result_large_err)]
```

Comment:
I think the root cause of this is that the `TokenRequest` error is large according to Clippy:

```
error: the `Err`-variant returned from this function is very large
   --> src/gcp/credential.rs:126:40
    |
86  | /     TokenRequest {
87  | |         source: crate::client::retry::RetryError,
88  | |     },
    | |_____- the largest variant contains at least 192 bytes
...
126 |       pub fn from_pem(encoded: &[u8]) -> Result<Self> {
    |                                          ^^^^^^^^^^^^

```

We could probably box the `Error::TokenRequest` variant and make clippy happier, but that would be a breaking API change. For now, let's leave the code as is an tell clippy it is ok